lumigator/python/mzai/sdk/tests/data/job.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
let's add these to a `static` resources folder like in evaluator to keep things consistent in the module https://github.com/mozilla-ai/lumigator/tree/main/lumigator/python/mzai/evaluator/tests/resources

Reply:
Is there anything else needed apart from changing the name of the directory?